sample/src/server.ts

```diff
@@ -10,4 +10,4 @@ async function start(port: number) {
   });
 }
 
-start(8020);
+start(8025);
```

Comment:
Any reason for this?

```suggestion
start(8020);
```

Reply:
local testing only. I'm indifferent